src/flownet/config_parser/_config_parser.py

```diff
@@ -136,6 +136,51 @@ def _to_abs_path(path: Optional[str]) -> str:
                                                     },
                                                 },
                                             },
+                                            "WOPT": {
```

Comment:
When adding WOPT (or WGPT or WWPT), should we allow for also adding WOPR (or WGPR or WWPR)? Conditioning on both rates and cumulatives i guess is kind of a messy thing with all the correlation going on. Should we add a check in the config_parser to make sure both are not defined?

Reply:
I will make a new issue suggesting this and add an item to the CHANGELOG